.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+### Summary
+
+This pull request aims to... (add your summary here)
+
+### Motivation / Context
+
+Please fill in any additional context. E.g. why should what you describe above be changed?
+
+### Are you introducing a breaking change?
```

Comment:
I suggest to make this a little less personal, otherwise it might sound like an interrogation
```suggestion
### Does this pull request introduce a breaking change?
```

Reply:
Interesting and good point! 👍